r_markdown/ef_summary.Rmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "Emission Factor Summary"
+output:
+  html_document:
+    toc: true
+    toc_float: true
+    theme: united
+---
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  library(tidyverse)
+  library(forcats)
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  source("../r_scripts/R_functions.R")
+```
+
+## Load and tidy data
```

Comment:
use one # for top level menu items (because of this your TOC isn't quite working as you'd expect)